packages/strapi-plugin-content-manager/admin/src/components/RelationPreviewList/RelationPreviewTooltip.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect, useCallback, useRef, useLayoutEffect } from 'react';
+import { Text, Padded } from '@buffetjs/core';
+import { LoadingIndicator } from 'strapi-helper-plugin';
+import PropTypes from 'prop-types';
+
+import getRequestUrl from '../../utils/getRequestUrl';
+import getMockData from './mockData';
+import { useListView } from '../../hooks';
+import Tooltip from '../Tooltip';
+
+const RelationPreviewTooltip = ({ tooltipId, rowId, mainField, name }) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [relationData, setRelationData] = useState([]);
+  const { slug } = useListView();
+  const tooltipRef = useRef();
+
+  const fetchRelationData = useCallback(async () => {
+    const requestURL = getRequestUrl(`collection-types/${slug}/${rowId}/${name}`);
```

Comment:
Can you try to add the maximum needed information into the layout? In the editview the layout regarding the relations includes the following:

```
layouts: {
  editRelations: [
    {
      fieldSchema: {},
      metadatas: {},
      name: 'categories',
      queryInfos: {
        endPoint: '/content-manager/relations/application::address.address/categories',
        containsKey: 'published_at_contains',
        defaultParams: {},
      },
    },
  ],
},
```

I think that using the same format for the relation, might help us later with i18n as we will have a more granular control of the endpoint, especially if we want to change the endpoint.

Here I would like us to generate the `queryInfos` info object for the relations (so you need to update the `formatLayout` and maybe the `formatLayoutToApi` utils) so you can add the static part of the endpoint `collection-types/${slug}/`. By doing so, you will also remove the dependency to the `useListView` hook.

Reply:
I don't really understand why we need the `containsKey: published_at_containes` here?